docs/dev/json.rst

```diff
@@ -175,3 +175,62 @@ submitted via the ``additional_info`` element. We're open to see how people
 will make use of these unspecified fields and may decide to formally specify or
 scrub elements in the future.
 
+Artist Statistics JSON
+----------------------
+The JSON documents returned from our API look like the following::
+
+   {
+     "payload": {
+        "user_id": "-- the MusicBrainz ID of the user --",
```

Comment:
Is this the string musicbrainz id or the database int id? If it's the string, can we call it musicbrainz_id

Reply:
It is username, that is string Musicbrainz ID of the user